scrapers/Xslist.yml

```diff
@@ -0,0 +1,75 @@
+name: "XsList (JAV)"
+performerByName:
+  action: scrapeXPath
+  queryURL: https://xslist.org/search?query={}&lg=en
+  scraper: performerSearch
+performerByURL:
+  - action: scrapeXPath
+    url: 
+      - xslist.org/en/model/
+    scraper: performerScraper
+
+xPathScrapers:
+  performerSearch:
+    performer:
+      Name: //li[@class="clearfix"]/h3/a/@title
+      URL: 
+        selector: //li[@class="clearfix"]/h3/a/@href
+  
+  performerScraper:
+    performer:
+      Name:
+        selector: //span[@itemprop="name"]/text()
+        # To have same name as JavLibrary so the scraper will found the performer.
```

Comment:
you mean same name convention as JavLibrary ? `Surname Name`   as opposed to `Name Surname` that is now?
I would rephrase it as 
`#Uncomment  below to convert to Surname Name (JavLibrary compatible)`

Reply:
Thanks for correct me, English is not my native language so sometimes my explanation is bad 👍